pkg/cloud/services/s3/s3_test.go

```diff
@@ -694,7 +694,7 @@ func TestDeleteObject(t *testing.T) {
 		}
 	})
 
-	t.Run("succeeds_when_bucket_has_already_been_removed", func(t *testing.T) {
+	t.Run("succeeds_when", func(t *testing.T) {
```

Comment:
Looks like there's another mock call that needs to be added. https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-aws/4879/pull-cluster-api-provider-aws-test/1770098021896294400#

Reply:
Weird, that test is passing when I run it: 
```
--- PASS: TestDeleteObject (0.00s)
    --- PASS: TestDeleteObject/is_idempotent (0.01s)
    --- PASS: TestDeleteObject/returns_error_when (0.00s)
        --- PASS: TestDeleteObject/returns_error_when/given_empty_machine_scope (0.00s)
        --- PASS: TestDeleteObject/returns_error_when/bucket_management_is_disabled_clusterwide (0.00s)
        --- PASS: TestDeleteObject/returns_error_when/object_deletion_fails (0.00s)
    --- PASS: TestDeleteObject/succeeds_when (0.01s)
        --- PASS: TestDeleteObject/succeeds_when/bucket_has_already_been_removed (0.00s)
        --- PASS: TestDeleteObject/succeeds_when/bucket_or_object_not_found (0.00s)
        --- PASS: TestDeleteObject/succeeds_when/object_has_already_been_removed (0.00s)
    --- PASS: TestDeleteObject/for_machine (0.02s)
        --- PASS: TestDeleteObject/for_machine/use_configured_bucket_name_on_cluster_level (0.00s)
        --- PASS: TestDeleteObject/for_machine/use_machine_role_and_machine_name_as_key (0.00s)
```

Reply:
Ok, it's fixed now.